SayTheirNames/Source/Controller/Home/Person/Views/PersonNewsTableViewCell.swift

```diff
@@ -150,8 +158,20 @@ extension PersonNewsTableViewCell: UICollectionViewDelegate, UICollectionViewDat
     
     // Set the data for each cell (color and color name)
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: cellType.identifier, for: indexPath)
-        return cell
+        switch cellType {
+        case .medias:
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: PersonMediaCollectionViewCell.reuseIdentifier,
```

Comment:
If we use this convenience method, we wont have to force unwrap the cell
```suggestion
            let cell: PersonMediaCollectionViewCell = collectionView.dequeueCell(for: indexPath)
```